### PR TITLE
Added notes for DLBs, VPCs, apps

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

**Review by MichelleZMY — DISMISSED** [review]

File: modules/ROOT/pages/_partials/no-rename.adoc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+//NO RENAME SHARED
+//tag::noRenameDLB[]
+[IMPORTANT]
```
Comment (kbostrom) [inline modules/ROOT/pages/_partials/no-rename.adoc:3]: I'm assuming that previous text clarifies DLB?

File: modules/ROOT/pages/_partials/no-rename.adoc [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+You cannot change the name of a DLB after you create it. To change the name, delete and recreate the DLB.
```
Comment (kbostrom) [inline modules/ROOT/pages/_partials/no-rename.adoc]:
```suggestion
You cannot change the name of a DLB after you create it. To change the name, delete and re-create the DLB.
```
OR
```suggestion
If you want to change the name of a DLB after you create it, you must delete and re-create it using the new name.

File: modules/ROOT/pages/_partials/no-rename.adoc [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+// end::noRenameDLB[]
+
+//tag::noRenameApp[]
+[IMPORTANT]
+You cannot change the name of an app after you deploy it. To change the name, delete and redeploy the app.
```
Comment (kbostrom) [inline modules/ROOT/pages/_partials/no-rename.adoc]: ditto above idea

**Review by kbostrom — DISMISSED** [review]
Edited

**Review by kbostrom — APPROVED** [review]
Reapproved

**Pull request merged**